android/BOINC/app/src/main/java/edu/berkeley/boinc/attach/ProjectAttachService.java

```diff
@@ -524,7 +526,18 @@ else if(statusCredentials.error_num != BOINCErrors.ERR_OK) {
                 }
                 switch(statusCredentials.error_num) {
                     case BOINCErrors.ERR_DB_NOT_UNIQUE:
+                        Handler handler = new Handler(Looper.getMainLooper());
+
+                        handler.post(new Runnable() {
+
+                            @Override
+                            public void run() {
+                                Toast.makeText(getApplicationContext(), "Wrong email or password", Toast.LENGTH_SHORT).show();
+                                return;
+                            }
+                        });
                         result = RESULT_NAME_NOT_UNIQUE;
+
                         return RESULT_NAME_NOT_UNIQUE;
                     case BOINCErrors.ERR_BAD_PASSWD:
```

Comment:
This error and all below should show corresponding message to the user too.
Please look at the errors and their description:
https://github.com/BOINC/boinc/blob/466e05dfd7e452373b80de89f096a1c477c0a8e8/android/BOINC/app/src/main/java/edu/berkeley/boinc/attach/ProjectAttachService.java#L409-L415